aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -222,25 +222,11 @@ Tensor to_functional_tensor(const Tensor& tensor) {
   TORCH_INTERNAL_ASSERT_DEBUG_ONLY(!isFunctionalTensor(tensor));
   return at::detail::make_tensor<FunctionalTensorWrapper>(tensor);
 }
-c10::List<Tensor> to_functional_tensor(const c10::List<Tensor>& t_list) {
-  c10::List<Tensor> outputs;
+std::vector<Tensor> to_functional_tensor(ITensorListRef t_list) {
```

Comment:
You aren't going to need to convert this back into a c10::List to stuff in an IValue, I hope?